week-1/L-functions-nested/exercise2.js

```diff
@@ -0,0 +1,21 @@
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+
+function uppercase(name) {
+  return name.toUpperCase();
+}
+
+function shoutyGreeting(name) {
```

Comment:
Really good work on giving the functions "speaking" names! 😀👏🏻

Reply:
Thanks.